test/Microsoft.AspNetCore.Razor.Runtime.Test/Runtime/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -2172,6 +2172,37 @@ public void CreateDescriptors_BuildsDescriptorsFromSimpleTypes()
                 TagHelperAttributeDescriptorComparer.Default);
         }
 
+        public static TheoryData<string, string> HtmlConversionData
+        {
+            get
+            {
+                var theoryData = new TheoryData<string, string>
```

Comment:
For consistency, just do `return new TheoryData<string, string>` instead of setting it to a variable.